src/h3lib/include/alloc.h

```diff
@@ -33,10 +33,10 @@
 extern "C" {
 #endif
 
-void* H3_MEMORY(malloc)(size_t size);
-void* H3_MEMORY(calloc)(size_t num, size_t size);
-void* H3_MEMORY(realloc)(void* ptr, size_t size);
-void H3_MEMORY(free)(void* ptr);
+void *H3_MEMORY(malloc)(size_t size);
+void *H3_MEMORY(calloc)(size_t num, size_t size);
+void *H3_MEMORY(realloc)(void *ptr, size_t size);
+void H3_MEMORY(free)(void *ptr);
```

Comment:
Why wasn't this previously formatted via clang?

Reply:
It was not included in the list of source files, presumably an oversight.